### PR TITLE
[16] Format logs for logit

### PR DESCRIPTION
**Pull request by darokel**
### Context

- https://trello.com/c/dd1KANnI/16-s-format-logs-for-logit

### Changes proposed in this pull request

- Setups logging for logit with semantic logger similarly to other DFE repos

### Guidance to review

- Fire up the rails server, assert logs are displayed based on semantic logger

### CODE REVIEW

**Comment by darokel** [comment]
[[S] Format logs for logit](https://trello.com/c/dd1KANnI/16-s-format-logs-for-logit)

File: config/initializers/semantic_logger.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  def format_backtrace
+    if hash[:message]&.start_with?("/")
+      message_lines = hash[:message].split("\n")
+      if message_lines.all? { |line| line.start_with?("/") }
+        hash[:backtrace] = hash[:message]
+        hash[:message] = "Exception occured: #{message_lines.first}"
+      end
+    end
+  end
+end
```
Comment (misaka) [inline config/initializers/semantic_logger.rb:55]: Do we want to enable this already for the production environment?

Comment (darokel) [reply config/initializers/semantic_logger.rb:55]: Yeah, it might be worth checking as a smoke test to see if it's logging the way we expect

**Review by defong — APPROVED** [review]

File: config/initializers/semantic_logger.rb (hunk 1 above)
Comment (misaka) [reply config/initializers/semantic_logger.rb:55]: I think my point is that we need this section:

https://github.com/DFE-Digital/teacher-training-api/blob/master/config/initializers/semantic_logger.rb#L51-L66

Comment (darokel) [reply config/initializers/semantic_logger.rb:55]: Done in https://github.com/DFE-Digital/register-trainee-teacher-data/pull/3/commits/7552f05a8a21009c15fcfd6ac6b24aa00cb8b440

**Review by misaka — APPROVED** [review]
:shipit:

**Pull request merged**